moderngl_window/context/sdl2/window.py

```diff
@@ -255,6 +255,9 @@ def process_events(self) -> None:
                 ):
                     self.close()
 
+                if self._fs_key is not None and event.key.keysym.sym == self._fs_key:

```

Comment:
Needs to check if `sdl2.SDL_KEYDOWN` as well or we'll only be in fs mode while the F11 so pressed down